make/mosquitto/patches/020-no_TIMESTAMP.patch

```diff
@@ -1,50 +0,0 @@
---- config.mk
```

Comment:
The https://github.com/Freetz/freetz/blob/master/make/mosquitto/patches/020-no_TIMESTAMP.patch is no longer required, because upstream removed the static TIMESTAMP inclusion at compile time. The static VERSION string in the binary now only consists of the version number ("1.6.8" in this case).

For reference see https://github.com/eclipse/mosquitto/blob/e55f7facce7628b33e57d6b44cc8e9dd1042e624/config.mk#L145, https://github.com/eclipse/mosquitto/blob/e55f7facce7628b33e57d6b44cc8e9dd1042e624/src/conf.c#L384 and https://github.com/eclipse/mosquitto/commit/2283585e391a6ca40a4abc252ac0e2111942b2d1.